### DISCUSSION TITLE
Why there is no official dashboard docker image?

### DISCUSSION THREAD

**Discussion by pitkes22** [mirumee/saleor-dashboard#1024] [RFC / Ideas] [answered]
I think it would be great for convenience to have an official dashboard docker image in this repository. It will in my opinion speed up the deployment process and future updates. Is there any reason why there is no such image being built?

**Comment by krzysztofwolski** [comment, marked as answer]
There is no image, because we suggest deploying the dashboard as a static app.
In the future, we would like to have a docker image which would handle both static (production) mode and dev mode, but there's no date on that.
